include/sundials/sundials_nonlinearsolver.h

```diff
@@ -47,7 +47,10 @@
 #ifndef _SUNNONLINEARSOLVER_H
 #define _SUNNONLINEARSOLVER_H
 
+#include <sundials/sundials_config.h>
 #include <sundials/sundials_context.h>
+#include <sundials/sundials_errors.h>
+#include <sundials/sundials_linearsolver.h>
```

Comment:
I'm surprised that this needed to be added here.  I'll add a suggestion to remove in case it is not actually needed.
```suggestion
```